sites/partners/lib/helpers.ts

```diff
@@ -144,6 +144,7 @@ export const createTime = (
   date: Date,
   formTime: { hours: string; minutes: string; period: TimeFieldPeriod }
 ) => {
+  if (!date || !formTime) return null
```

Comment:
is `null` the most appropriate value here?

Reply:
I thought so. Returning a valid Date if you provide garbage seemed like it could lead to hard-to-track-down bugs. But lmk if you have a different suggestion.

Reply:
SGTM; I also wasn't sure whether `undefined` was possible to return or if throwing an exception was better; I don't have strong opinions, just wondering.